src/runway/blueprints/staticsite/staticsite.py

```diff
@@ -198,7 +410,13 @@ def create_template(self):
             )
         )
 
-        cfdirectoryindexrewriterole = template.add_resource(
+    def add_index_rewrite_role(self):
+        """Add an index rewrite role to the template.
+
+        Return:
+            dict: The index rewrite role
+        """
+        return self.template.add_resource(
             iam.Role(
                 'CFDirectoryIndexRewriteRole',
                 Condition='DirectoryIndexSpecified',
```

Comment:
Might as well make this the 'CFEnabledAndDirectoryIndexSpecified' condition as well